torch/utils/dlpack.py

```diff
@@ -3,28 +3,30 @@
 import torch
 import enum
 
-from torch._C import _from_dlpack
 from torch._C import _to_dlpack as to_dlpack
 
 __all__ = [
     "DLDeviceType",
     "from_dlpack",
-    "to_dlpack",
 ]
 
-
 class DLDeviceType(enum.IntEnum):
     # Enums as in DLPack specification (aten/src/ATen/dlpack.h)
     kDLCPU = 1,
-    kDLGPU = 2,
-    kDLCPUPinned = 3,
+    kDLCUDA = 2,
```

Comment:
These are BC-breaking right?

Reply:
Yes. But, this is needed if we want to follow the enum in `aten/src/ATen/dlpack.h`. This change was actually supposed to happen in #65047 (when the enum names actually changed).